merlin-server/sql/merlin/tables/constraint.sql

```diff
@@ -83,23 +83,20 @@ before update on "constraint"
 for each row
 execute function constraint_set_updated_at();
 
-create function constraint_check_constraint_run()
-returns trigger
-security definer
-language plpgsql as $$begin
-  if new.definition != old.definition then
-    update constraint_run
-    set definition_outdated = true
-    where constraint_id = new.id
-      and constraint_definition != new.definition
-      and definition_outdated = false;
-  else
-    update constraint_run
-    set definition_outdated = false
-    where constraint_id = new.id
-      and constraint_definition == new.definition
-      and definition_outdated = true;
-  end if;
+create or replace function constraint_check_constraint_run()
```

Comment:
Minor: This should be just `create` here.
```suggestion
create function constraint_check_constraint_run()
```

Reply:
Got it, I only updated `constraint.sql` and keep `up/down.sql` the same as It made sense to me to `create or replace` the migration file.

Reply:
Yup, that was exactly what I meant 🚀